src/OpenApi/Model/Paths.php

```diff
@@ -31,6 +56,9 @@ public function getPath(string $path): ?PathItem
 
     public function getPaths(): array
     {
+        // sort paths by tags, then by path for each tag
+        uksort($this->paths, $this->comparePathsByKey(...));
+
```

Comment:
maybe that this should be done inside the OpenApi Normalizer instead of the Model class?

Reply:
That's a good point, i wasn't sure where this is used and did it as a quick fix. 

I'll dig a bit further then.